deployments/cardano-prod.nix

```diff
@@ -6,13 +6,24 @@ let
       ./../modules/datadog.nix
       ./../modules/papertrail.nix
     ];
-    services.dd-agent.tags = ["env:production"];
 
     # DEVOPS-64: disable log bursting
     services.journald.rateLimitBurst = 0;
 
     # Initial block is big enough to hold 3 months of transactions
     deployment.ec2.ebsInitialRootDiskSize = mkForce 700;
+
+    services.dd-agent.tags = ["env:production"];
+    services.dd-agent.processConfig = ''
+      init_config:
+
+      instances:
+      - name: cardano-node
+        search_string: ['cardano-node']
+        exact_match: False
+        thresholds:
+          critical: [1, 2]
+    '';
```

Comment:
Yes, this is confusing.

We will have to unify nixpkgs commit across projects as that will assure better reproducibility. Moreover, our Hydra should build against one nixpkgs commit (or at least as few as possible).

Unfortunately, this isn't as easy as moving things around, as there are many ways to pin nixpkgs (in shell, or in Nix, or in `config.yaml`, etc).

At the very least we should track where it's all defined so we can make reason and simplify.

In this very PR I have linked the nixpkgs pull request that this change was tested with as datadog changes aren't merged upstream yet.